### ISSUE TITLE
"on_key_down" event in Windows is documented badly

### ISSUE THREAD

**Issue by janssen** [kivy/kivy#4137]
There's an extra undocumented argument, which seems to be a list of the modifier keys.

**Comment by abject** [comment]
The argument `something_undocumented` seems to be an integer.

Furthermore `keycode` is not any more a list of the keycode value and text, it's only the keycode integer value.

A simple example showing this is:
``` python
from kivy.app import App
from kivy.uix.widget import Widget
from kivy.core.window import Window


class KeyDown(App):
    def build(self):
        Window.bind(on_key_down=self.key_action)
        return Widget()

    def key_action(self, *args):
        print("got a key event: %s" % list(args))


if __name__ == '__main__':
    KeyDown().run()
```

When pressing on a key you will some message like: 
```
got a key event: [<kivy.core.window.window_sdl2.WindowSDL object at 0x7f4831c012b8>, 113, 4, 'q', []]
```
for eg when I pressed on 'q' key.
or:
```
got a key event: [<kivy.core.window.window_sdl2.WindowSDL object at 0x7fb6a7e1d2b8>, 305, 224, 'ı', []]
got a key event: [<kivy.core.window.window_sdl2.WindowSDL object at 0x7fb6a7e1d2b8>, 113, 4, 'q', ['ctrl']]
```
 for Ctrl+q combo.

I cant' find any example in the documentation about the third argument passed by the event. All examples have only the following arguments: `keyboard, keycode, text, modifiers`.

So if I changed the previous example by:
```python
from kivy.app import App
from kivy.uix.widget import Widget
from kivy.core.window import Window


class KeyDown(App):
    def build(self):
        Window.bind(on_key_down=self.key_action)
        return Widget()

    def key_action(self, keyboard, keycode, text, modifiers):
        print("got a key event: %s" % list(args))


if __name__ == '__main__':
    KeyDown().run()
```
When I run it I get the following error:
```python
 Traceback (most recent call last):
   File "test.py", line 16, in <module>
     KeyDown().run()
   File "/usr/lib/python3/dist-packages/kivy/app.py", line 828, in run
     runTouchApp()
   File "/usr/lib/python3/dist-packages/kivy/base.py", line 504, in runTouchApp
     EventLoop.window.mainloop()
   File "/usr/lib/python3/dist-packages/kivy/core/window/window_sdl2.py", line 663, in mainloop
     self._mainloop()
   File "/usr/lib/python3/dist-packages/kivy/core/window/window_sdl2.py", line 602, in _mainloop
     self.modifiers):
   File "kivy/_event.pyx", line 714, in kivy._event.EventDispatcher.dispatch (kivy/_event.c:7681)
   File "kivy/_event.pyx", line 1225, in kivy._event.EventObservers.dispatch (kivy/_event.c:13524)
   File "kivy/_event.pyx", line 1149, in kivy._event.EventObservers._dispatch (kivy/_event.c:13050)
 TypeError: key_action() takes 5 positional arguments but 6 were given
```

Here some information about the environment:
* **OS:** Xubuntu 16.04.4 LTS
* **Python**: 3.5.2
* **Kivy:** 1.10.0

**Comment by mgc8** [comment]
Same exact issue as described above with the latest version of Kivy (2.0.0) on Linux/Kubuntu 20.04.

**Comment by ElliotGarbus** [comment]
@Julian-O This issue can be closed.  The modifiers list is documented.  https://kivy.org/doc/stable/api-kivy.core.window.html#kivy.core.window.Keyboard

**Comment by Julian-O** [comment]
Fixed by #4446
